core/src/avm1/globals/text_field.rs

```diff
@@ -353,6 +355,25 @@ pub fn set_html_text<'gc>(
     Ok(())
 }
 
+pub fn background_color<'gc>(
+    this: EditText<'gc>,
+    _activation: &mut Activation<'_, 'gc, '_>,
+) -> Result<Value<'gc>, Error<'gc>> {
+    Ok(this.background_color().into())
+}
+
+pub fn set_background_color<'gc>(
+    this: EditText<'gc>,
+    activation: &mut Activation<'_, 'gc, '_>,
+    value: Value<'gc>,
+) -> Result<(), Error<'gc>> {
+    if let Ok(rgb) = value.coerce_to_u32(activation) {
+        this.set_background_color(activation.context.gc_context, rgb & 0xFFFFFF);
+        // TODO: check if can be transparent
+    }
+    Ok(())
```

Comment:
Instead of matching for `Ok`, we should toss the error upwards with `?` in the wacko case of that the parameter has a throwing `valueOf`:
```
var o = {
	valueOf: function() {
		trace("valueOf");
		throw "BAD";
	}
};
txt.borderColor = o;
```

Reply:
Ok, thanks. That makes sense

Reply:
I've made those changes. However, I did a similar thing in `set_text_color` in #1878. Should I create a new pr and change that?

Reply:
Thanks! You can do it in this PR if you'd like, sorry I missed it on the earlier PR.

Reply:
Done :)